lti_consumer/api.py

```diff
@@ -180,29 +190,65 @@ def get_lti_1p3_content_url(config_id=None, block=None, hint=""):
         b. Deep Linking is configured, but a single ltiResourceLink was selected.
     2. The Deep Linking content presentation URL if there's more than one
        Lti DL content in the database.
+
+    The launch_information parameter is a dictionary that contains data necessary and related to an LTI 1.3 launch.
+    It provides a mechanism for the context to provide contextually defined or stored launch information to the library.
+    The schema of launch_information may change as we support additional claims or LTI specifications.
+
+    Below are the currently support keys for launch_information.
+
+    config_id (required; string): the config_id field of an LtiConfiguration to use for the launch
+    user_role (required; string): the user's role as one of the keys in LTI_1P3_ROLE_MAP: staff, instructor, student,
+        or guest.
+    user_id (required; string, int): the user's unique identifier
```

Comment:
specifically the LTI-facing identifier right, not our own ID?

Reply:
It can be whatever identifier the user of the library will use to uniquely identify the user. Ideally, we wouldn't use our internal database identifier, yes, but each user may define that value independently.